tools/build-on-openbsd

```diff
@@ -2,6 +2,12 @@
 
 fail() { echo "FAILED:" "$@" 1>&2; exit 1; }
 
+PYTHON=${PYTHON:-python3}
+if [ ! $(which ${PYTHON}) ]; then
```

Comment:
Which shell is `/bin/sh` on openbsd? Typically `command -v` is more portable.

Reply:
sh is Bourne Shell.
(the default (root) shell of ksh)